src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.csproj

```diff
@@ -48,6 +48,53 @@
     <Reference Include="FSharp.Compiler.CodeDom">
       <HintPath>..\..\packages\FSharp.Compiler.CodeDom.1.0.0.1\lib\net40\FSharp.Compiler.CodeDom.dll</HintPath>
     </Reference>
+    <Reference Include="Newtonsoft.Json">
+      <HintPath>..\..\packages\Newtonsoft.Json.11.0.1\lib\net45\Newtonsoft.Json.dll</HintPath>
+    </Reference>
+    <Reference Include="NuGet.Frameworks">
+      <HintPath>..\..\packages\NuGet.Frameworks.4.6.0\lib\net46\NuGet.Frameworks.dll</HintPath>
```

Comment:
These packages are what need to be added to `src/Xamarin.Android.Build.Tasks.tpnitems`. If they need to be redistributed, they need to be in *a* `.tpnitems` file, with corresponding license information.

Reply:
done. Don't think we need to add `System.Runtime` or `System.Runtime.InteropServices` since I don't think we need to ship those as they should be resolved as part of the .net runtime.. in theory.